mods/ts/mod.yaml

```diff
@@ -270,12 +271,21 @@ ModContent:
 			Download: basefiles
 		music: Base Game Music
 			TestFiles: ^Content/ts/scores.mix
-			Sources: tibsun, tibsun-linux, tfd, origin
+			Sources: tibsun, tibsun-linux, tfd, origin, firestorm
 			Download: music
+		expansion: Firestorm Expansion
+			TestFiles: ^Content/ts/expand01.mix
+			Sources: firestorm, tfd
+			Required: true
```

Comment:
Because we now have two `Required: true` packages we now need a separate `ts-quickinstall` download that includes all the required content.